.github/workflows/markdown-links.yml

```diff
@@ -0,0 +1,15 @@
+name: Check Broken Markdown Links
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Would you mind replicating this in the main workflow to see if it solves the problem please?

Reply:
The build is failing because it does not find the `.circle/config.yml` you deleted in this https://github.com/frgfm/PyroNear/pull/68 
Maybe, in first place, one should tell github to stop looking for this config since it's been deleted (https://circleci.com/docs/2.0/enable-checks/) ?

I don't see why moving the new markdown action to main file would change anything, it is actually well working, as you can see in the checks.

Reply:
Yes, I intend to do so. The only issue is that if I stop now to tell github to stop looking for CircleCI config, PRs that were open before the changes will lose CI checks (the action is for the whole repo).

Sorry, I wasn't very clear: I meant keeping your separate workflow, but in order to check if changing the trigger event to pull_request fixes the builds, would you mind changing additionally, in the main workflow, the triggering event? (as I did in the last merged PR)

Reply:
Actually, nevermind, would you mind just making any additional commit, I believe it will trigger the workflows! (cf. #69 where somme commits were made after the changes and the workflows correctly trigger)

Reply:
The triggering event is already set on `pull_request` as [it is also done in the last merged PR](https://github.com/frgfm/PyroNear/pull/73/commits/ada63052baea4915853463e5194f37e056fa37f7)

Reply:
The fix you mentioned was merged after the last commit you in this PR until my previous comment. And apparently, that was it, since your last commit triggered the workflows :)

Aso, to make sure this also run, once the commits are squash-merged into master, I'd suggest to add a commit on master as trigger event (like in #74)

Let me know if you run into another trouble!

Reply:
You should realize that the required tests are all passing in the current PR. I won't add any fake commit to trigger any more CI, that's not a good way to contribute to code base.

Please realize as I did for your PR (https://github.com/pyronear/PyroNear/pull/73) that if all required tests are passing we are ready to merge.